modules/linear_boltzmann_solvers/executors/pi_keigen_scdsa.cc

```diff
@@ -176,17 +176,8 @@ PowerIterationKEigenSCDSA::Execute()
     auto Sf0_ell = CopyOnlyPhi0(front_gs_, q_moments_local_);
 
     // This solves the inners for transport
+    auto phi0_lph_i = CopyOnlyPhi0(front_gs_, phi_old_local_);
```

Comment:
I know the variable had that name before this PR, but what does it mean? Can a comment be added?

Reply:
Modified notation and added comments.